src/app/containers/Caption/index.jsx

```diff
@@ -9,22 +9,32 @@ import InlineLink from '../InlineLink';
 
 const componentsToRender = { fragment: Fragment, urlLink: InlineLink };
 
-const renderText = block => {
-  const textBlocks = block.model.blocks[0].model.blocks[0].model.blocks;
+const renderText = textBlocks => (
+  <Blocks blocks={textBlocks} componentsToRender={componentsToRender} />
+);
 
-  return <Blocks blocks={textBlocks} componentsToRender={componentsToRender} />;
-};
+const renderCaption = (block, imageCaptionOffscreenText) => (
+  <Caption>
+    {imageCaptionOffscreenText ? (
+      <VisuallyHiddenText>{imageCaptionOffscreenText}</VisuallyHiddenText>
+    ) : null}
+    {renderText(block)}
+  </Caption>
+);
+
+const renderMultipleCaptions = (blocks, imageCaptionOffscreenText) =>
+  blocks.map(block =>
+    renderCaption(block.model.blocks, imageCaptionOffscreenText),
+  );
 
 const CaptionContainer = ({ block }) => (
   <ServiceContext.Consumer>
-    {({ imageCaptionOffscreenText }) => (
-      <Caption>
-        {imageCaptionOffscreenText ? (
-          <VisuallyHiddenText>{imageCaptionOffscreenText}</VisuallyHiddenText>
-        ) : null}
-        {renderText(block)}
-      </Caption>
-    )}
+    {({ imageCaptionOffscreenText }) =>
+      renderMultipleCaptions(
+        block.model.blocks[0].model.blocks,
```

Comment:
There's a [deepGet utility that you can use](https://github.com/bbc/simorgh/blob/latest/src/app/helpers/json/deepGet.js), that is a safer way to access deeply nested items in JSON. 
```suggestion
        deepGet(['model', 'blocks', [0], 'model', 'blocks'], block)
```

Reply:
I think its `0` instead of `[0]`, but yay for safe nested objects :D 🎉